app/src/main/java/org/mozilla/focus/fragment/WebFragment.java

```diff
@@ -107,11 +111,25 @@ public void onPause() {
             webView.saveWebViewState(session);
         }
 
+        if (AppConstants.isGeckoBuild() && dayPassedSinceLastUpload()) {
+            // Record new edited date
+            PreferenceManager.getDefaultSharedPreferences(getContext()).edit()
+                    .putLong(lastMobileMetricsPing, System.currentTimeMillis()).apply();
+            webView.getTelemetrySnapshots();
```

Comment:
The naming here is confusing: get***() implies that this method returns something. We could either make it return the JSON data and we construct the ping here or we rename it to something like sendTelemetrySnapshots()